web/modules/weather_blocks/src/Plugin/Block/Test/EndToEnd/DailyForecast/DailyForecast.php.test

```diff
@@ -260,4 +260,24 @@ final class DailyForecastStructureTest extends EndToEndBase
 
         $this->assertEquals($expected, $actual["today"]);
     }
+
+    /**
+     * Ensure that we set the flag for only low temp after 6pm
+     *
+     * @group e2e
+     * @group eric
+     */
+    public function testOnlyLowAfter6pmToday()
+    {
+        $now = \DateTimeImmutable::createFromFormat(
+            \DateTimeInterface::ISO8601_EXPANDED,
+            "2024-05-07T18:01:00-0500",
+        );
+
+        $this->onLocationRoute(32.778, -96.796);
```

Comment:
I had to think about this for a minute, but I get it now. Based on how the `useOnlyLowForToday` flag gets sets below, I was thinking the only thing that mattered was how the data was returned from the API, but then I remembered that we subdivide the daily forecast periods into days based on the current time and the times of the periods.

https://github.com/weather-gov/weather.gov/pull/1319/files#diff-3cf5ed10dc8559478c4d8c2552ad2691ced84edeb99636aa49fdeebdacec014cR262